cryptography/src/lib.rs

```diff
@@ -394,51 +408,97 @@ mod tests {
     }
 
     #[test]
-    fn test_secp256r1_validate() {
+    fn test_secp256r1_base_validate() {
```

Comment:
dumb idea: can we just call these `secp256r1`? Do we need to append `_base`?

Reply:
I like the idea (in this case) of having a default (non-recoverable) and then having `_recoverable`.